spec/features/saml/loa1_sso_spec.rb

```diff
@@ -41,17 +41,17 @@
       expect(page).to_not have_css('.accordion-header')
     end
 
-    it 'allows user to view recovery code via profile', :js do
+    it 'user cannot view recovery code via profile', :js do
```

Comment:
the "via profile" part of the description confuses me? The goal is to disable it during signup, not from the profile page right?